GOVERNANCE.md

```diff
@@ -1,3 +1,246 @@
+the Steering Committee invites your feedback there.
+
+## Committee members
+
+Seats on the Steering Committee are held by an individual, not by their
+employer.
+
+The current membership of the committee is currently (listed alphabetically by
+first name):
+
+| &nbsp;                                                         | Member           | Organization |
+| -------------------------------------------------------------- | ---------------- | ------------ |
+
+[TODO: Pending elections]
+
+## Decision process
+
+The Steering Committee desires to always reach consensus.
+
+Decisions requiring a vote include: issuing written policy, amending existing
+written policy, creating, removing, or modifying a committee or SIG, all spending,
+hiring, and contracting, official responses to publicly raised issues, or any
+other decisions that at least two of the members present decide require a vote.
+
+Decisions are made in meetings when a quorum of more than half of the members
+are present (and all members have been informed of the meeting), and may pass
+with more than half the members of the committee supporting it.
+
+## Getting in touch
+
+Open an issue on [the governance repo](https://github.com/Kuadrant/governance) and indicate that you would like
+attention from the steering committee.
+
+## Composition
+
+The steering committee has 3 seats. These seats are
+open to any project contributor. See
+[candidate eligibility](#candidate-eligibility) for a definition.
+
+## Election Procedure
+
+### Timeline
+
+Steering Committee elections are held annually. Six weeks or more before the
+election, the Steering Committee will appoint Election Officer(s) (see below).
+Four weeks or more before the election, the Election Officer(s) will issue a
+call for nominations, publish the list of voters, and open the call for
+exceptions. Three days before the election the call for nominations and exceptions
+will be closed. The election will be open for voting not less than two weeks and
+not more than four. The results of the election will be announced within one
+week of closing the election. New Steering Committee members will take office on
+October 1st of each year.
+
+### Election Officer(s)
+
+Six weeks or more before the election, the Steering Committee will appoint
+between one and three Election Officer(s) to administer the election. Elections
+Officers will be community members in good standing who are eligible to
+vote, are not running for Steering in that election, who are not currently part
+of the Steering Committee and can make a public promise of impartiality. They
+will be responsible for:
+
+- Making all announcements associated with the election
+- Preparing and distributing electronic ballots
+- Judging exception requests
+- Assisting candidates in preparing and sharing statements
+- Tallying voting results according to the rules in this charter
+
+### Eligibility to Vote
+
+Anyone who has at least 10 contributions in the last 12 months is
```

Comment:
This says 10 contributions, but the threshold for "contributors" is 50 contributions.  It'll make your life easier if both have the same threshold.

Reply:
Good catch.
I'll leave this comment thread open as a reminder to sanity check numbers across the doc before anything gets merged